CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ### Bug fixes
 
+- Do not pass empty `-o` parameter to squashfuse_ll to avoid `unknown
+  option` error.
```

Comment:
Thank you for the entry, but please make it more user focused.  For example
```
- Avoid `unknown option` error when using a bare squashfs image with an unpatched `squashfuse_ll`.
```

Reply:
Yes, that's better. I wasn't sure about the circumstances, whether there are other cases with an empty optsStr.